typescript/infra/scripts/check-validator-announce.ts

```diff
@@ -0,0 +1,41 @@
+import { defaultMultisigConfigs } from '@hyperlane-xyz/sdk';
+import { eqAddress } from '@hyperlane-xyz/utils';
+
+import { isEthereumProtocolChain } from '../src/utils/utils.js';
+
+import { getArgs, withChains } from './agent-utils.js';
+import { getEnvironmentConfig, getHyperlaneCore } from './core-utils.js';
+
+async function main() {
```

Comment:
Could be nice to extend this to check is the validators have setup their S3 bukcet correctly and are signing the latest check point https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/cab86f2f971c2c03f7620101134f6fb2f06690c5/typescript/cli/src/validator/preFlightCheck.ts#L51

Reply:
I should probably refactor the code so it can be reused

Reply:
there's already a script for that - but I wanted a very specific announce-only check for now

Reply:
what I'd like really is the CLI command for checking validators, but to be able to operate that over the set of default multisigs we track

Reply:
see existing script: 
- https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/330d302a6181191e61141786c0999d7200ab6da4/typescript/infra/scripts/verify-validators.ts#L33
- https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/330d302a6181191e61141786c0999d7200ab6da4/typescript/infra/src/agents/aws/validator.ts#L48

which iterates through all storage locations onchain and also checks their checkpoint/mailbox/etc